splunklogger.js

```diff
@@ -436,7 +440,7 @@ SplunkLogger.prototype._sendEvents = function(context, callback) {
     requestOptions.headers["Authorization"] = "Splunk " + this.config.token;
     // Manually set the content-type header, the default is application/json
     // since json is set to true.
-    requestOptions.headers["Content-Type"] = "application/x-www-form-urlencoded";
```

Comment:
not the best coding practice here haha. this change you made is much better. so we'll just have to update references to the splunklog constructor to add the contentType param?